sdk/nodejs/package.json

```diff
@@ -25,6 +25,7 @@
     },
     "pulumi": {
         "resource": true,
-        "name":"astra","server": "https://github.com/pulumiverse/pulumi-astra/releases/download/${VERSION}"
+        "name": "astra",
+        "server": "https://github.com/pulumiverse/pulumi-astra/releases/download/${VERSION}"
```

Comment:
Can you also incorporate the new support for Github Download URLs?

https://www.pulumi.com/docs/guides/pulumi-packages/how-to-author/#support-for-github-releases

Yes, this requires Pulumi CLI 3.35.3 as a minimum, but since this version, the CLI really speaks to the Github API, and it can pick the right version. That `${VERSION}` placeholder is no longer needed.

I documented here where to set this in `provider/resources.go`:
https://github.com/pulumiverse/infra/blob/main/README.md#package-registries